web/modules/custom/pineapple_helper/pineapple_helper.links.menu.yml

```diff
@@ -0,0 +1,6 @@
+pineapple_helper.main:
+  title: Report an Issue
+  description: 'Report an issue'
+  route_name: pineapple_helper.main
+  weight: 10
+  parent: system.admin
```

Comment:
Add options here that include the ajax class added on the button and also include the data attribute added on that button. Examples of formatting should be found on other modules or in the "array"

Reply:
done